### PR TITLE
Fix date timestamp typo ( Created and Modified date are different for newly saved Image Preview)

### LINKED ISSUE #1708: Created and Modified date are different for newly saved Image Preview
### Preconditions

1. Install Magento with Adobe Stock Integration
2. Configured integration in Stores -> Configuration -> Advanced-> System -> Adobe Stock Integration fieldset
3. Enhanced Media Gallery Enabled
 
### Steps to reproduce (*)
1. Go to **Content - Media Gallery**
2. Click **Search Adobe Stock** and Save an image Preview
3. Select the previously saved preview image
4. Click "three dots", select **View Details**
5. Observe the _Created_ and _Modified_ dates

### Expected result (*)
The _Created_ and _Modified_ dates should be equal

### Actual result (*)
The _Created_ date is 7 hours earlier than _Modified_ date
time_zone

### NOTE
The same happens with locally uploaded images after Edit.
1. Upload an image
2. Check that the Modified And Created are equal
3. Edit image Description and Save

AR: The _Created_ date is 7 hours earlier than _Modified_ date

### PR DESCRIPTION
**Pull request by Nazar65**
### Fixed Issues (if relevant)

1. Fixes https://github.com/magento/adobe-stock-integration/issues/1708: Created and Modified date are different for newly saved Image Preview

### Manual testing scenarios (*)
1. Go to **Content - Media Gallery**
2. Click **Search Adobe Stock** and Save an image Preview
3. Select the previously saved preview image
4. Click "three dots", select **View Details**
5. Observe the _Created_ and _Modified_ dates

The _Created_ and _Modified_ dates should be equal

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

File: app/code/Magento/MediaGalleryUi/Test/Mftf/ActionGroup/AdminEnhancedMediaGalleryVerifyImageCreatedAtNotEqualsUpdatedAtTimeActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminEnhancedMediaGalleryVerifyImageCreatedAtNotEqualsUpdatedAtTimeActionGroup">
```
Comment (rogyar) [inline app/code/Magento/MediaGalleryUi/Test/Mftf/ActionGroup/AdminEnhancedMediaGalleryVerifyImageCreatedAtNotEqualsUpdatedAtTimeActionGroup.xml]: Please, use the "Assert" prefix for similar checks. So we have consistent naming for such cases. I would suggest the following name for the action group

`AssertAdminEnhancedMediaGalleryImageCreatedAtNotEqualsUpdatedAtTimeActionGroup`

File: app/code/Magento/MediaGalleryUi/Test/Mftf/ActionGroup/AdminEnhancedMediaGalleryVerifyUploadedImageDateTimeEqualsActionGroup.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminEnhancedMediaGalleryVerifyUploadedImageDateTimeEqualsActionGroup">
```
Comment (rogyar) [inline app/code/Magento/MediaGalleryUi/Test/Mftf/ActionGroup/AdminEnhancedMediaGalleryVerifyUploadedImageDateTimeEqualsActionGroup.xml]: Please, use the "AssertAdmin" prefix in the same way as it was mentioned in the previous comment.

**Review by rogyar — CHANGES_REQUESTED** [review]

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-7988](https://jira.corp.magento.com/browse/ENGCOM-7988) has been created to process this Pull Request

**Comment by engcom-Lima** [comment]
:heavy_check_mark: QA Passed
The Created and Modified dates are equal
Screenshot_75
Created and Modified dates are displayed correctly after editing for locally uploaded images
Screenshot_76
Manual testing [scenario](https://studio.cucumber.io/projects/131313/test-plan/folders/1320712/scenarios/3961351)

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-7988](https://jira.corp.magento.com/browse/ENGCOM-7988) has been created to process this Pull Request

**Comment by engcom-Lima** [comment]
:heavy_check_mark: QA Passed
The Created and Modified dates are equal
Screenshot_99
Created and Modified dates are displayed correctly after editing 
Screenshot_98
See depended [comment](https://github.com/magento/magento2/pull/29452#issuecomment-671957509)

**Pull request merged**